test/embulk/guess/test_query_string.rb

```diff
@@ -0,0 +1,54 @@
+require "prepare_embulk"
+require "embulk/guess/query-string"
+require "embulk/data_source"
+
+module Embulk
+  module Guess
+    class QueryStringPluginTest < Test::Unit::TestCase
+      class TestGuessLines < self
+        def test_parse_line_without_options
+          actual = QueryString.new.guess_lines(config, sample_lines_1)
+          expected = {
+            "parser" => {
+              type: "query-string",
+              schema: [
+                {name: "foo", type: :long},
+                {name: "bar", type: :string},
+                {name: "baz", type: :string},
+              ]
+            }
+          }
+          assert_equal(expected, actual)
+        end
+      end
+
+      private
+
+      def sample_lines_1
```

Comment:
Hmm..I'll think more meaningful names:) (I want each name to specify content of data)